src/tribol/mesh/CouplingScheme.hpp

```diff
@@ -333,6 +333,12 @@ class CouplingScheme
                              const integer cycle, 
                              const real t );
 
+  void setLoggingLevel( const LoggingLevel log_level ) { m_loggingLevel = log_level; }
```

Comment:
Please add doxygen comments for these methods

Reply:
Added.